sw/device/tests/aon_timer_wdog_bite_reset_test.c

```diff
@@ -0,0 +1,104 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/freestanding/limits.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_aon_timer.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/test_framework/ottf.h"
+#include "sw/device/lib/testing/test_framework/test_status.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+// TODO:(lowrisc/opentitan#XXXX) Move these difinitions to somewhere else?
```

Comment:
Nit: D**e**finitions? Also, can you open an issue about this and tag it here? Or just manually plonk them in `dif_rstmgr.h` for now?

Reply:
I'd suggest you make that change here and wait for review. Otherwise you can end up in a situation where no-one does anything because they don't realise you're waiting for them to reply.

Reply:
Done!